charts/launch-agent/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: launch-agent
 icon: https://em-content.zobj.net/thumbs/240/apple/354/rocket_1f680.png
 description: A Helm chart for running the W&B Launch Agent in Kubernetes
 type: application
-version: 0.11.0
+version: 0.12.0
```

Comment:
What are the rules for minor vs patch update here?

Reply:
arbitrary. I'm actually going to do this as a patch